test/e2e/singlecluster/e2e_test.go

```diff
@@ -490,6 +493,85 @@ var _ = ginkgo.Describe("Kueue", func() {
 			})
 		})
 	})
+	ginkgo.When("Creating a JobSet", func() {
+		var (
+			defaultRf    *kueue.ResourceFlavor
+			localQueue   *kueue.LocalQueue
+			clusterQueue *kueue.ClusterQueue
+		)
+		ginkgo.BeforeEach(func() {
+			defaultRf = testing.MakeResourceFlavor("default").Obj()
+			gomega.Expect(k8sClient.Create(ctx, defaultRf)).Should(gomega.Succeed())
+			clusterQueue = testing.MakeClusterQueue("cluster-queue").
+				ResourceGroup(
+					*testing.MakeFlavorQuotas(defaultRf.Name).
+						Resource(corev1.ResourceCPU, "2").
+						Resource(corev1.ResourceMemory, "2G").Obj()).Obj()
+			gomega.Expect(k8sClient.Create(ctx, clusterQueue)).Should(gomega.Succeed())
+			localQueue = testing.MakeLocalQueue("main", ns.Name).ClusterQueue("cluster-queue").Obj()
+			gomega.Expect(k8sClient.Create(ctx, localQueue)).Should(gomega.Succeed())
+		})
+		ginkgo.AfterEach(func() {
+			gomega.Expect(util.DeleteLocalQueue(ctx, k8sClient, localQueue)).Should(gomega.Succeed())
+			gomega.Expect(util.DeleteAllJobsetsInNamespace(ctx, k8sClient, ns)).Should(gomega.Succeed())
+			util.ExpectClusterQueueToBeDeleted(ctx, k8sClient, clusterQueue, true)
+			util.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, defaultRf, true)
+		})
+
+		ginkgo.It("Should run a jobSet if admitted", func() {
+			jobSet := testingjobset.MakeJobSet("job-set", ns.Name).
+				Queue("main").
+				ReplicatedJobs(
+					testingjobset.ReplicatedJobRequirements{
+						Name:        "replicated-job-1",
+						Replicas:    2,
+						Parallelism: 2,
+						Completions: 2,
+						Image:       "gcr.io/k8s-staging-perf-tests/sleep:v0.1.0",
+						// Give it the time to be observed Active in the live status update step.
+						Args: []string{"5s"},
```

Comment:
I would suggest to use 1ms container runtime. I think it is enough to wait for the JobSet to be finished in this test. For multicluster we verify updates, but live updates to the status are a feature on its own.

Reply:
thanks for pointing that out. done.